mbuild/utils/conversion.py

```diff
@@ -1,25 +1,143 @@
 """mBuild conversion utilities."""
+from warnings import warn
+
 import numpy as np
 
 
-def RB_to_OPLS(c0, c1, c2, c3, c4, c5):
-    """Convert Ryckaert-Bellemans type dihedrals to OPLS type.
+def RB_to_OPLS(c0, c1, c2, c3, c4, c5, error_tol=1e-4):
```

Comment:
The only thing I can think of that might be an alternate way to do this conversion would be to use an `assert_opls_f0_is_0` or some similarly named flag. The default value would be True and the conversion would error out and say, your f0 parameter is not 0. Then, you have the option to change the flag to False if you don't care that it isn't 0.

Reply:
@CalCraven for the RB_to_OPLS function, I agree with your statement with the default being assert_opls_f0_is_0=True.  This would make the user acknowledge that is potentially not a perfect conversion when they integrate the function into their code.  

For now, we can set LAMMPS and HOOMD to the ignore if f0 not zero (assert_opls_f0_is_0=false).  I will change it if @justinGilmer and @chrisiacovella are in agreement.

Reply:
This is completed